lib/download.js

```diff
@@ -0,0 +1,37 @@
+import path from "path";
+import fs from "fs/promises";
+import * as core from "@actions/core";
+import * as tc from "@actions/tool-cache";
+import { getOsTriplet } from "./platform";
+import { versionWithPrefix } from "./versions";
+
+export async function downloadSnForge(repo, version) {
```

Comment:
technically, this function downloads starknet-foundry (so snforge + sncast) and not only snforge. Let's adjust the function name.

Reply:
the same applied to log messages/variables etc in this function